doc/topics/releases/version_numbers.rst

```diff
@@ -46,3 +46,50 @@ to be cut and the month is ``June``. This would make the new release number
 
 After the release is cut, new features would be worked on under the ``Xenon``
 code name and the process repeats itself.
+
+
+Version numbers, Git and salt --version
+-------
+The salt version, for programmers, is based on `git describe` and presented to end-users with `salt --version`.
+
+Arguments for `git checkout`
+  +------------+----------------------------------------------------------------------------+
+  |  Argument  |                                           Comment                          |
+  +============+============================================================================+
+  | v2016.11   | Tag of the moment that the 2016.11 branch was created from develop         |
+  +------------+----------------------------------------------------------------------------+
+  | 2016.11    | **Release branch** Actively developed bug-fixes for all 2016.11.* releases |
+  +------------+----------------------------------------------------------------------------+
+  | v2016.11.1 | Tag of the moment that the 2016.11.1 branch was created                    |
```

Comment:
Change the word `branch` to `release` here. The `v2016.11.1` syntax is not a branch. It is a tag signaling the commit that a release is based on.